.github/workflows/test.yaml

```diff
@@ -7,14 +7,11 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.8]
     steps:
       - uses: actions/checkout@v2
-      - name: Build and start containers
+      - name: Start service containers
         run: make svc-up
       - name: Wait until containers are ready
-        run: sleep 10
+        run: sleep 10 && timeout 60 docker-compose exec -T -- wes-rabbitmq rabbitmqctl await_startup && sleep 10
```

Comment:
did you find scenarios where rabbitmq was not ready after the initial 10s wait?  i am sorta wondering why we need the first `sleep 10` now.

Reply:
The short answer is - after adding the delays, I haven't experienced a problem. The two delays were to address:

First - it turns out rabbitmqctl can fail with an error because rabbitmq is not ready to accepts commands...
Second - it takes a moment for rabbitmq to load the definitions.json file, so ever after starting, we need a short delay for rabbitmq to be functionally ready.

I can try to reduce these / find a more slick way, but for the purposes of testing, this seems to work well enough for now.

Reply:
all good. leave em.